src/App.css

```diff
@@ -2,13 +2,14 @@ body {
   margin: 0;
   font-size: 1rem;
   font-weight: 800;
-  background: linear-gradient(rgb(6, 6, 46), rgb(63, 11, 11));
+
+  /* background: linear-gradient(rgb(6, 6, 46), rgb(63, 11, 11)); */
```

Comment:
- [ ] Please kindly delete the commented line of code, it's very important to have a clean code base 🙃 🙃